src/components/pages/teams/hero/hero.jsx

```diff
@@ -7,13 +7,13 @@ import GitHubIcon from '../../../../icons/github.inline.svg';
 import DiscordIcon from './images/discord.inline.svg';
 import TwitterIcon from './images/twitter.inline.svg';
 
+import MergedPrs from 'components/pages/team/hero/merged-prs';
```

Comment:
Please remove this line from here and add before GithubIcon, make sure to add a empty line after this as well.